Python/pathconfig.c

```diff
@@ -734,8 +734,7 @@ _PyPathConfig_ComputeSysPath0(const PyWideStringList *argv, PyObject **path0_p)
 /* Search for a prefix value in an environment file (pyvenv.cfg).
```

Comment:
It doesn't search "for a prefix". It searchs for the the line "home=<path>" and return `<path>`. It tolerates whitespaces around "=".